interface/wx/dataview.h

```diff
@@ -1926,6 +1926,25 @@ class wxDataViewRenderer : public wxObject
     virtual bool SetValue(const wxVariant& value) = 0;
 
     /**
+        Set the transformer object to be used to customize values before they
+        are rendered.
+
+        Can be used to change the value if it is shown on a highlighted row
+        (i.e. in selection) which typically has dark background. It is useful
+        in combination with wxDataViewTextRenderer with markup and can be used
+        e.g. to remove background color attributes inside selection, as a
+        lightweight alternative to implementing an entire
+        wxDataViewCustomRenderer specialization.
+
```

Comment:
This might be obvious, but I'd add a sentence saying that `@a transformer can be @NULL to reset any transformer currently being used`.